config/docker-compose.yml.example

```diff
@@ -71,6 +71,15 @@ services:
     volumes:
       - /tmp:/tmp:ro
 
+  shodan:
+    build:
+      context: ../src/main/scala/org/holmesprocessing/totem/services/shodan
+      args:
+        conf: ${CONFSTORAGE_SHODAN}service.conf
+    ports:
+      - "9730:8080"
+    restart: unless-stopped
+
```

Comment:
Should be up below PassiveTotal as the services are sorted by port, then by alphabet